app/src/main/res/layout/activity_leave_entry.xml

```diff
@@ -5,6 +5,355 @@
+                    android:textSize="16sp"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="@id/glReference"
+                    app:layout_constraintTop_toBottomOf="@id/tvTempRollNo" />
+
+                <TextView
+                    android:id="@+id/tvTempLeaveType"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="12dp"
+                    android:text="@string/casual"
+                    android:textColor="@color/text_color"
+                    android:textSize="16sp"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="@id/glReference"
+                    app:layout_constraintTop_toBottomOf="@id/tvTempDepartment" />
+
+                <androidx.constraintlayout.widget.Guideline
+                    android:id="@+id/glReference"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical"
+                    app:layout_constraintGuide_begin="116dp" />
+
+                <TextView
+                    android:id="@+id/tvLeave"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="16dp"
+                    android:fontFamily="@font/inter_semibold"
+                    android:text="@string/leave_"
+                    android:textColor="@color/black"
+                    android:textSize="20sp"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/tvLeaveType" />
+
+                <TextView
+                    android:id="@+id/tvFrom"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="16dp"
+                    android:fontFamily="@font/inter_semibold"
+                    android:text="@string/from"
+                    android:textColor="@color/black"
+                    android:textSize="16sp"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@id/tvLeave" />
+
+                <TextView
+                    android:id="@+id/tvFromDate"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="12dp"
+                    android:text="@string/_21_05_2024"
+                    android:textColor="@color/text_color"
+                    android:textSize="16sp"
+                    app:layout_constraintStart_toEndOf="@id/tvFrom"
+                    app:layout_constraintTop_toTopOf="@id/tvFrom" />
```

Comment:
Wrap them with a Linear layout, and keep the layout_gravity center. Do the same for tvToDate.